### PR TITLE
DecEmber announcement

### PR DESCRIPTION
**Pull request by MelSumner**
## What it does
Adds teaser content regarding the DecEmber learning team event.

### CODE REVIEW

**Comment by Alonski** [comment]
@MelSumner
This seems to fail because of this:
```69:77-69:81 warning Be careful with “fear”, it’s profane in some cases fear retext-profanities```
I'm on mobile so I can't fix right now :)

**Comment by MelSumner** [comment]
ok, I updated the content to change the phrasing. 

Also, we can add the label filter as the URL if it needs one, does that work?

File: source/blog/2018-11-16-the-ember-times-issue-73.md [hunk 1]
```diff
@@ -128,4 +128,4 @@ That's another wrap! ✨
 
 Be kind,
 
-Chris Ng, Jessica Jordan, Kenneth Larsen, Niels Rasmussen, Ryan Mark the crowd and the Learning Team
\ No newline at end of file
+Chris Ng, Jessica Jordan, Kenneth Larsen, Niels Rasmussen, Ryan Mark the crowd and the Learning Team
```
Comment (amyrlam) [inline source/blog/2018-11-16-the-ember-times-issue-73.md]:
```suggestion
Melanie Sumner, Chris Ng, Jessica Jordan, Kenneth Larsen, Niels Rasmussen, Ryan Mark the crowd and the Learning Team
```

File: source/blog/2018-11-16-the-ember-times-issue-73.md [hunk 2]
```diff
@@ -65,8 +65,8 @@ If you want to browse through all of the great talks from EmberFest, check out t
 
 ---
 
-## [SECTION TITLE](#section-url)
-
+## [DecEmber is almost here!](https://github.com/ember-learn/guides-source/labels/DecEmber)
```
Comment (amyrlam) [inline source/blog/2018-11-16-the-ember-times-issue-73.md]:
```suggestion
## [DecEmber is almost here! 📆](https://github.com/ember-learn/guides-source/labels/DecEmber)
```

File: source/blog/2018-11-16-the-ember-times-issue-73.md (hunk 1 above)
Comment (amyrlam) [reply source/blog/2018-11-16-the-ember-times-issue-73.md]: We should also add your name to L3 and add a short blurb to L12 e.g. "A teaser for Dec**Ember** 😄"...

**Pull request merged**